packages/generate-scaffold/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as c from "ansi-colors";
+import chalk from "chalk";
+import { exec } from "child_process";
+import * as fs from "fs";
+import * as fse from "fs-extra";
+import * as inquirer from "inquirer";
+import { resolve } from "path";
+
+const cl = console;
+const log = cl.log;
+
+const USER_DIRECTORY: string = process.env.PWD ? process.env.PWD : process.cwd();
+import * as QUESTIONS from "./utils/questions";
```

Comment:
imports on top when since it's scope is outer most

Reply:
`PROMPT_QUESTIONS`

Reply:
done